src/pages/algebraic-effects-for-the-rest-of-us/index.ja.md

```diff
@@ -0,0 +1,416 @@
+---
+title: 我々向けの Algebraic Effects 入門
+date: '2019-07-21'
+spoiler: ブリトーとは違うんですよ
```

Comment:
I first did not know what's the nuance of this ( a palody of https://neoeinstein.github.io/monads-are-not-burritos/#/ ? )